docker/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM debian:bullseye-slim
+
+WORKDIR /web
+
+COPY . .
+
+RUN mv .env.example .env \
```

Comment:
This should probably be split into multiple RUN commands based on semantics, like a run command to install deps, another to build the package / install python deps, another to clean build-only deps, another to do django stuff, another for permission stuff.

As it stands this is not very readable and also it doesn't allow Docker to cache things properly. E.g. if the deps stay the same and you only change some permission stuff later on, with multiple RUN commands Docker would rebuild much faster. Granted that during CI there is probably no cache anyway but still.

Reply:
Multiple `RUN` commands would make the image larger.
Although, `docker-slim` would probably still compress it to the same size. :thinking:

I thought about adding cache to the CI build but it should be fast enough anyway.

Reply:
I didn't know that multiple RUN commands inflate image size. Docker moment. I don't think we've got that many dependencies anyway so it's fine but if we did, then the only thing I'd change was just to decouple things into two phases: 
1. copy the dependency information run all dep related stuff
2. then copy everything else and do the rest of the thing

so you can at least cache dependency installation. In this specific case it's fine though